sql/report_queries/claimed_returned/claimed_returned.sql

```diff
@@ -1,272 +1,182 @@
-        (SELECT end_date :: VARCHAR FROM parameters) AS date_range,
-    -- circulation fields
-    sc.effective_location_at_checkout,
-    sc.loan_item_status,
-    sc.loan_action,
-    sc.renewal_count,
-    stl.loan_count_historical,
-    stl.renewal_count_historical,
-    sc.loan_date,
-    sc.loan_due_date,
-    sc.loan_return_date,
-    sc.loan_policy_name,
-    sc.claimed_returned_date,
-    -- inventory fields
-    si.title,
-    si.items_perm_location_name,
-    si.items_temp_location_name,
-    si.items_effective_location_name,
-    si.holdings_perm_location_name,
-    si.holdings_temp_location_name,
-    si.institution_name,
-    si.campus_name,
-    si.library_name,
-    si.barcode,
-    si.material_type,
-    si.item_call_number,
-    si.holdings_call_number,
-    si.volume,
-    si.enumeration,
-    si.chronology,
-    si.copy_number,
-    siwn.notes,
-    si.shelving_title,
-    siwpd.dates_of_publication,
-    si.cataloged_date,
-    -- user fields
-    su.first_name,
-    su.middle_name,
-    su.last_name,
-    su.email,
-    su.patron_group,
-    sup.first_name AS proxy_first_name,
-    sup.middle_name AS proxy_middle_name,
-    sup.last_name AS proxy_last_name,
-    sup.email AS proxy_email,
-    sup.patron_group AS proxy_patron_group
+    (
+        SELECT
+            start_date::VARCHAR
+        FROM
+            parameters) || ' to '::VARCHAR || (
+        SELECT
+            end_date::VARCHAR
+        FROM
+            parameters) AS date_range,
+    li.loan_id,
+    li.item_id,
+    li.loan_date,
+    li.loan_due_date,
+    li.loan_return_date,
+    json_extract_path_text(l.data, 'claimedReturnedDate')::DATE AS claimed_returned_date,
+    li.item_status,
+    nn.item_notes,
+    li.item_effective_location_name_at_check_out,
+    li.loan_policy_name,
+    li.permanent_loan_type_name,
+    li.current_item_temporary_location_name,
+    li.current_item_effective_location_name,
+    li.current_item_permanent_location_name,
+    li.current_item_permanent_location_library_name,
+    li.current_item_permanent_location_campus_name,
+    li.current_item_permanent_location_institution_name,
+    li.barcode,
+    li.material_type_name,
+    li.chronology,
+    li.copy_number,
+    li.enumeration,
+    li.item_level_call_number,
+    li.number_of_pieces,
+    ie.volume,
+    ie.call_number,
+    --he.holdings_id,
+    --he.instance_id,
+    he.permanent_location_name,
+    he.temporary_location_name,
+    he.shelving_title,
+    ii.cataloged_date,
+    pd.dates_of_publication,
+    lrc.num_loans,
+    lrc.num_renewals,
+    --li.user_id,
+    li.patron_group_name,
+    --  li.proxy_user_id,
+    json_extract_path_text(uu.data, 'personal', 'firstName')::VARCHAR AS first_name,
+    json_extract_path_text(uu.data, 'personal', 'middleName')::VARCHAR AS middle_name,
+    json_extract_path_text(uu.data, 'personal', 'lastName')::VARCHAR AS last_name,
+    json_extract_path_text(uu.data, 'personal', 'email'::VARCHAR) AS email
```

Comment:
Runs either way but right parenthesis looks misplaced.
... 'email')::VARCHAR AS email

Reply:
Yes, it is misplaced.

Reply:
The casts to varchar are not necessary because json_extract_path_text() essentially returns varchar.

Reply:
These are minor changes, but I think I will go ahead and make the suggested edits while working through the review.